CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ It includes many new rules, including all new techniques introduced in MITRE ATT
 - rule: add file-scope feature (`function-name`) for recognized library functions #567 @williballenthin
 - main: auto detect shellcode based on file extension #516 @mr-tz
 - main: more detailed progress bar output when matching functions #562 @mr-tz
+- explorer: add `1` argument to automatically analyze when running capa explorer #548 @Ana06
+  `ida_loader.load_and_run_plugin("capa_explorer", 1)`
```

Comment:
```suggestion
  `ida_loader.load_and_run_plugin("capa_explorer", Options.ANALYZE)`
```

Reply:
users would need to define `Options.ANALYZE` in their scripts/code and I think this is suppose to be an `int`. 😕

Reply:
ok or we can export it somehow? maybe put it in `capa/ida/__init__.py`?

Reply:
Plugins can also be configured in the `plugins.cfg` file. There it needs to be a number.
Ref: https://hex-rays.com/blog/scriptable-plugins/